libs/platform/src/lib/approval-flow/approval-flow.component.ts

```diff
@@ -249,6 +265,14 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
         this.nodeClick.emit(node);
     }
 
+    _onNodeAdd(node: ApprovalNode): void {
```

Comment:
please add typedoc
please check the whole file

Reply:
Done